bin/node/runtime/src/lib.rs

```diff
@@ -120,17 +120,9 @@ parameter_types! {
 	pub const BlockHashCount: BlockNumber = 250;
 	/// We allow for 2 seconds of compute with a 6 second average block time.
 	pub const MaximumBlockWeight: Weight = 2_000_000_000_000;
```

Comment:
```suggestion
	pub const MaximumBlockWeight: Weight = 2 * WEIGHTS_PER_SECOND;
```

Reply:
If you do this, then even if we decide to scale all the weights again (to for example 1 weight = 1 nano-seconds) then things will not break here again. 

If you want to make it even fancier, you could impl `ops::Add` and `ops::Mul` for `Get<_>` and then we can define the `MaximumBlockWeight ` inside `parameter_type` in `weights::constants` and then have it very nicely here or in any other runtime:
impl frame_system::Trait for Runtime {
     type MaximumBlockWeight = 2 * WeightUnitsPerSecond;
}
```

but a simple const is also already good enough.